### ISSUE TITLE
[feature] Secure boot images in web UI

### ISSUE THREAD

**Issue by jfroy** [siderolabs/omni#175]
### Problem Description

There does not appear to be a way to download secure boot images.

**Comment by Unix4ever** [comment]
Actually you can use CLI, like that:

```
omnictl download --secureboot iso
```

**Comment by jfroy** [comment]
> Actually you can use CLI, like that:
> 
> 
> 
> ```
> 
> omnictl download --secureboot iso
> 
> ```

Ah great. I hadn't tried the CLI before.

**Comment by Unix4ever** [comment]
@smira do you think we're ready to enable secure boot checkbox in the UI?

**Comment by smira** [comment]
> @smira do you think we're ready to enable secure boot checkbox in the UI?

yes, only for bare-metal for now, and make sure you link to the docs (https://www.talos.dev/v1.7/talos-guides/install/bare-metal-platforms/secureboot/), and this is Talos 1.5+ only.

**Comment by CappyT** [comment]
Is there any update on this? Also, once the iso from the command line is used, do upgrade from UI respect the secureboot status?

**Comment by Unix4ever** [comment]
> Is there any update on this? Also, once the iso from the command line is used, do upgrades from UI respect the secureboot status?

We should enable it, will prioritize it.

The upgrades should handle the secureboot.
